CSharp/JobPrep/Program.cs

```diff
@@ -0,0 +1,333 @@
+﻿#define VERBOSE
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Concurrent;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Microsoft.Azure.Batch;
+using Microsoft.Azure.Batch.Auth;
+using Microsoft.Azure.Batch.Common;
```

Comment:
Code analysis has a new rule that says you should do this. Surprisingly, resharper did complain about this but it's about scope as opposed to the way things are currently done.